pyproject.toml

```diff
@@ -0,0 +1,76 @@
+[tool.poetry]
+name = "pelican-pdf"
+version = "0.0.0"
+description = "The PDF Generator plugin automatically exports articles and pages as PDF files as part of the site generation process"
+authors = ["Pelican Dev Team <authors@getpelican.com>"]
+license = "AGPL-3.0"
+readme = "README.md"
+keywords = ['pelican', 'plugin', 'pdf', 'generator']
+repository = "https://github.com/pelican-plugins/pdf"
+documentation = "https://docs.getpelican.com"
+packages = [
+    { include = "pelican" },
+]
+
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Framework :: Pelican",
+    "Framework :: Pelican :: Plugins",
+    "Intended Audience :: End Users/Desktop",
+    "Operating System :: OS Independent",
+    "Topic :: Internet :: WWW/HTTP",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+
+[tool.poetry.urls]
+"Funding" = "https://donate.getpelican.com/"
+"Issue Tracker" = "https://github.com/pelican-plugins/pdf/issues"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+pelican = "^4.5"
+markdown = {version = "^3.2.2", optional = true}
+rst2pdf = "^0.98"
+xhtml2pdf = "^0.2.5"
+
+[tool.poetry.dev-dependencies]
+black = {version = "^19.10b0", allow-prereleases = true}
+flake8 = "^3.8"
+flake8-black = "^0.2.0"
+invoke = "^1.3"
+isort = "^5.4"
+livereload = "^2.6"
+markdown = "^3.2.2"
+pytest = "^6.0"
+pytest-cov = "^2.8"
+pytest-pythonpath = "^0.7.3"
+pytest-sugar = "^0.9.4"
+Werkzeug = "^1.0"
+
+[tool.poetry.extras]
+markdown = ["markdown"]
+
+[tool.autopub]
+project-name = "PDF Generator"
+git-username = "botpub"
+git-email = "botpub@autopub.rocks"
```

Comment:
[I just fixed a few things in the CookieCutter template.](https://github.com/getpelican/cookiecutter-pelican-plugin/commit/93509377a60082b846c41e21932b5f68a40bd3e0) Please add the following line to this section:
```toml
append-github-contributor = true
```